test/module.js

```diff
@@ -0,0 +1,134 @@
+var tests = {};
+
+var testsStats = {
+	failed: 0, skipped: 0, todo: 0
+};
+
+var assertionsStats = {
+	passed: 0, failed: 0
+};
+
+var done = false;
+
+QUnit.testDone( function( details ) {
```

Comment:
I don't think we need to collect all this information. We should only collect the test that ran and then any modifiers. So `testId`, `skipped`, or `todo`.